hugo-modules/core/utils/seo/private/get-data.html

```diff
@@ -96,63 +80,61 @@
 	{{ end }}
 {{ end }}
 
-
 {{/* Title
   ----------------------------
 	We use the following logic
 	1. Every pages: `{.Title} | {site.Title}`
```

Comment:
Regarding https://github.com/jungvonmatt/wekit/issues/17, we should either only use this scheme when seo_title is not explicitly set, or make it configurable with some kind of variable substitution. Maybe we could use the i18n function for that?
```
{{ i18n "title_schema" . }}
```

Reply:
Good point. But I don't want to make this PR any bigger. Let's just start a "clean up templates" next after this content model clean up. We can address other topics regarding SEO. For example #22, #21, ... and probably more.